lib/index.js

```diff
@@ -192,6 +192,8 @@ class BundleTrackerPlugin {
         fileInfo.path = path.relative(this.outputChunkDir, fileInfo.path);
       }
 
+      fileInfo.sourceFilename = stats.compilation.assetsInfo.get(assetName).sourceFilename;
```

Comment:
Have you tried replacing `stats.compilation.assetsInfo.get(assetName)` with `file.assetName`? Or even the `assetName` that's already in scope here? I thought you previous commit was de-structuring file into assetName.

Reply:
> Have you tried replacing stats.compilation.assetsInfo.get(assetName) with file.assetName?

I'm not sure if I got the question.

`stats.compilation.assetsInfo.get(assetName)` is actually a Map that contains the key `sourceFilename`

Reply:
Thanks for the review! muito obrigado